### PR TITLE
Removed create stylesheets and added a custom theme

### PR DESCRIPTION
**Pull request by markusamuel**

### CODE REVIEW

File: android/settings.gradle [hunk 1]
```diff
@@ -1,3 +1,5 @@
 rootProject.name = 'metremenqeemi'
+include ':react-native-vector-icons'
```
Comment (mark-yacoub) [inline android/settings.gradle]: did u test on android?

File: ios/Podfile [hunk 2]
```diff
@@ -12,6 +12,8 @@ target 'metremenqeemi' do
     :hermes_enabled => false
   )
 
+  pod 'RNVectorIcons', :path => '../node_modules/react-native-vector-icons'
```
Comment (mark-yacoub) [inline ios/Podfile]: did u test on ios?

File: package.json [hunk 3]
```diff
@@ -23,11 +23,14 @@
     "prop-types": "^15.8.1",
     "react": "17.0.2",
     "react-native": "0.66.4",
+    "react-native-cli": "^2.0.1",
```
Comment (mark-yacoub) [inline package.json]: why

File: package.json [hunk 4]
```diff
@@ -23,11 +23,14 @@
...
     "react-native-gesture-handler": "^1.10.3",
+    "react-native-paper": "^4.11.2",
     "react-native-progress-bar-animated": "^1.0.6",
     "react-native-reanimated": "^1.13.2",
     "react-native-safe-area-context": "^3.3.2",
     "react-native-screens": "^3.10.1",
+    "react-native-vector-icons": "^9.1.0",
```
Comment (mark-yacoub) [inline package.json]: where is this used?

File: src/App.js [hunk 5]
```diff
@@ -29,17 +30,40 @@ const About = ({navigation}) => (
 
 const Drawer = createDrawerNavigator();
 
+const theme = {
+  ...DefaultTheme,
+  dark: false,
+  colors: {
+    ...DefaultTheme.colors,
+
```
Comment (mark-yacoub) [inline src/App.js]: remove space

File: src/App.js [hunk 6]
```diff
@@ -29,17 +30,40 @@ const About = ({navigation}) => (
...
+    accent: '#FEC203',
+    gold: '#FEC203',
+  },
+
+  seperator: {
+    height: 1,
+    width: '86%',
+    marginLeft: '5%',
+  },
+  fontSize: 18,
```
Comment (mark-yacoub) [inline src/App.js]: order them alphabetically

File: src/screens/LessonsList.js [hunk 7]
```diff
@@ -9,21 +10,30 @@ const lessonsList = {
...
+        >
+          {lessonName}
+        </Text>
       </TouchableOpacity>
     );
   };
 
   return (
-    <View style={styles.container}>
+    <View style={{flex: 1}}>
```
Comment (mark-yacoub) [inline src/screens/LessonsList.js]: isn't RN `flex:1` by default?

File: src/screens/level1/learningLetters.js [hunk 8]
```diff
@@ -104,8 +105,8 @@ const Level1LearningLetters = ({lessonNumber}) => {
   };
 
   return (
-    <SafeAreaView style={styles.container}>
-      <ScrollView style={styles.scrollView}>
+    <SafeAreaView style={{flex: 1}}>
```
Comment (mark-yacoub) [inline src/screens/level1/learningLetters.js]: ditto

**Review by mark-yacoub — CHANGES_REQUESTED** [review]
you seem to have broken soomething with yarn.lock aw 7aga. 
when i pull your branch and `yarn`, i get this
```
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
info There appears to be trouble with your network connection. Retrying...
info There appears to be trouble with your network connection. Retrying...
info There appears to be trouble with your network connection. Retrying...
```

compared with `neeb` in so many ways and it's def only on your branch

File: src/screens/LessonsList.js (hunk 7 above)
Comment (markusamuel) [reply src/screens/LessonsList.js]: Good catch

**Pull request closed without merging**